analytics/find-hds-react-version-usage.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Call to Github API to find all occurrences of "hds-react" under City-of-Helsinki org.
+curl -H "Accept: application/vnd.github+json"  -H "Authorization: token $TOKEN" "https://api.github.com/search/code?q=hds-react+in:file+filename:package.json+org:City-of-Helsinki&per_page=100" | jq '. | .total_count as $total_count | { total_count: $total_count, repositories: [.items[] | {name: .repository.name, url: .repository.html_url, package_url: .git_url, path: .path}]}' > hds-react-version-usage.json
```

Comment:
Could this be combined with the curl in the search-repos script? That way it would easier to develop and fix later?